zerver/lib/management.py

```diff
@@ -137,9 +139,22 @@ def get_users(
                 return user_profiles
 
         if options["users"] is None:
-            return []
+            return UserProfile.objects.none()
         emails = {email.strip() for email in options["users"].split(",")}
-        return [self.get_user(email, realm) for email in emails]
+        # This is inefficient, but we fetch (and throw away) the
+        # get_user of each email, so that we know that the email
+        # address/realm returned only one result.
```

Comment:
Hmm, under what circumstance could it return more than one result?

Reply:
See `get_user`, below -- if you provide an email but not a realm, and the email exists in multiple realms.

Reply:
Extended the comment slightly to clarify this.